src/components/presentational/Button.css

```
@@ -0,0 +1,27 @@
+.button {
+    padding: 8px 35px;
+    background: linear-gradient(94.66deg, #4498d984 17.46%, #7367beb9 87.77%);
+    color: rgba(255, 255, 255, 1);
+    border-radius: 25px;
+    border: none;
+    font-family: Cormorant;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: center;
+    box-shadow: 7px 6px 28px 1px rgba(0, 0, 0, 0.24); 
+    transition: opacity 0.6s;
+    
+}
+
+.button:disabled {
+    opacity: 0.5;
+    background: linear-gradient(94.66deg, #8bacc684 17.46%, #928cb7b9 87.77%);
+}
+
+.button:active:not(:disabled),
+.link:active {
```

```suggestion
 {
```

закрался партизан

стили `.link` должны быть в своё CSS файле, если мы говорим про компонентный подход

можешь создать компонент `Link`

вижу в `Header.css` ещё стили `.link`